src/prefect/server/api/server.py

```diff
@@ -210,12 +210,17 @@ def replace_placeholder_string_in_files(
 def copy_directory(directory, path):
```

Comment:
Made a few changes to this method:
1. Cleans up some of the variable names to make parsing the method easier
2. Removes subdirectories when walking the file graph and recreates them (`copy2` does this with files by default)